etcd/etcd-live-cluster-reconfiguration.md

```diff
@@ -101,40 +101,34 @@ member 60e8a32b09dc91f1 is healthy: got healthy result from http://10.0.1.3:2379
 cluster is healthy
 ```
 
-We have to run `etcdctl` from a working node, or just use [`ETCDCTL_ENDPOINT`][etcdctl-endpoint] environment variable or parameter to specify `etcdctl` to use healthy member node.
+Run `etcdctl` from a working node, or use the [`ETCDCTL_ENDPOINT`][etcdctl-endpoint] environment variable or command line option to point `etcdctl` at any healthy member node.
 
-Then we have to [remove failed][etcdctl-member-remove] 10.0.1.2 member from your etcd cluster. This will tell all the other nodes in the cluster that a human has determined this node is dead and not to connect to it ever again:
+[Remove the failed member][etcdctl-member-remove] `10.0.1.2` from the etcd cluster. The remove subcommand informs all other cluster nodes that a human has determined this node is dead and not available for connections:
 
 ```sh
 $ etcdctl member remove 1609b5a3a078c227
 Removed member 1609b5a3a078c227 from cluster
 ```
 
-Then on the failed node (10.0.1.2) we have to stop etcd2 service:
+Then, on the failed node (`10.0.1.2`), stop the etcd2 service:
 
 ```sh
 $ sudo systemctl stop etcd2
 ```
 
-And clean up the `/var/lib/etcd2` directory:
+Clean up the `/var/lib/etcd2` directory:
 
 ```sh
 $ sudo rm -rf /var/lib/etcd2/*
 ```
 
-You have to make sure that this directory exists and empty. It you removed this directory accidentally, you can recreate it with the command below:
+Check that the `/var/lib/etcd2/` directory exists and is empty. If you removed this directory accidentally, you can recreate it with the proper modes by using:
 
 ```sh
 $ sudo systemd-tmpfiles --create /usr/lib64/tmpfiles.d/etcd2.conf
 ```
 
-or with this one:
-
-```sh
-$ sudo install -m 755 -d -o etcd -g etcd /var/lib/etcd2
```

Comment:
The goal of the edit was to present one way to do it, rather than two. But you make a fair point -- the invocation of `install` shows the desired mode on the directory. Of course, so would, e.g., `mkdir -p /var/lib/etcd2 && chown -R etcd:etcd !$ && chmod 0755 !$`

So I would like to choose one "official" way -- and I think the systemd utility approach is the most consistent with our "way" of doing service management.

Now that I have explained my thinking, do you think we still need both?